pkg/ccl/backupccl/targets.go

```diff
@@ -380,33 +381,55 @@ func checkMissingIntroducedSpans(
 			tablesIntroduced[descpb.ID(tableID)] = struct{}{}
 		}
 
+		requiredIntroduction := func(table *descpb.TableDescriptor) error {
+			if _, required := requiredTables[table.GetID()]; !required {
+				return nil
+			}
+
+			if !includeTableSpans(table) {
+				return nil
+			}
+
+			_, inPrevBackup := prevTables[table.GetID()]
+
```

Comment:
minor subjective calls:  I'd go ahead and just if/early return on each of these as well for consistency with the other two and then unconditional err return at end (or make all 4 just defs and then one if block).

One advantage of the early all returns is that you could comment why each case makes it OK, e.g.
```
if _, inPre...; !inPrev {
  // If it was in previous backup, then it does not need to be introduced since...
return nil
}
```
And just purely reading-comprehension, I'd do `if _, introduced` first, since if it is introduced anyway, there's no need to see if it needs to be introduced, but that's just me.

Reply:
I like this idea! Done.